property.go

```diff
@@ -194,11 +254,14 @@ func parsePropertyParam(r *BaseProperty, contentLine string, p int) (*BaseProper
 	k, v := "", ""
 	k = string(contentLine[p : p+tokenPos[1]])
 	p += tokenPos[1]
+	if p >= len(contentLine) {
+		return nil, p, fmt.Errorf("missing property param operator for %s in %s", k, r.IANAToken)
+	}
 	switch rune(contentLine[p]) {
 	case '=':
 		p += 1
 	default:
-		return nil, p, fmt.Errorf("missing property value for %s in %s", k, r.IANAToken)
+		return nil, p, fmt.Errorf("%s for %s in %s", MissingPropertyValueError, k, r.IANAToken)
```

Comment:
As per the other one. Multiple `%w` are supported as of 1.20.. 

I guess that means we should push the go.mod up to that. -- Based on this I'm in support. 1.20 is less than 2 years old. But most stacks are rather current.

Reply:
Ah this one is just missing a `%w` wish github had the same feature gitlab does "suggest a fix"